cpp/oneapi/dal/backend/primitives/element_wise.hpp

```diff
@@ -55,13 +55,70 @@ inline sycl::event element_wise(sycl::queue queue,
     });
 }
 
+template <typename Functor,
+          typename Input1Type,
+          typename Input2Type,
+          typename OutputType,
+          ndorder input1_layout,
+          ndorder input2_layout,
+          ndorder output_layout>
+inline sycl::event element_wise(sycl::queue& queue,
```

Comment:
All those functions do approximately the same thing. It would be better to have at least 1 description at the beginning of the file and a short comment describing the specifics of each implementation near the declaration.

Reply:
Okay. We have many primitives. Not all of them have comments. I won't fix it all in one PR. Sorry

We need to plan this activity ahead